js/kucoinfutures.js

```diff
@@ -1059,11 +1064,32 @@ module.exports = class kucoinfutures extends kucoin {
             'leverage': 1,
         };
         const stopPrice = this.safeValue2 (params, 'triggerPrice', 'stopPrice');
+        const stopLossPrice = this.safeValue (params, 'stopLossPrice');
+        const takeProfitPrice = this.safeValue (params, 'takeProfitPrice');
+        const isStopLoss = stopLossPrice !== undefined;
+        const isTakeProfit = takeProfitPrice !== undefined;
+        const stop = this.safeString (params, 'stop');
+        const stopPriceType = this.safeString (params, 'stopPriceType', 'MP');
```

Comment:
This also isn't needed, you can just set a default value of `MP` for `stopPriceType`, and then if the user passes `stopPriceType` to params, then the request parameter will be overwritten